### DISCUSSION TITLE
Override default fzfx window settings

### DISCUSSION THREAD

**Discussion by damanis** [linrongbin16/fzfx.nvim#705] [Q&A]
I use the following code to change window split to horizontal.
It worked before but in last versions using it causes problem: it is impossible to use `--` to specify additional `rg` arguments. What is not correct in this configuration code?
```lua
  local fzfx = require("fzfx")

  fzfx.setup({
    live_grep = {
        fzf_opts = {
          { "--prompt",         "Buffers > " },
          { "--delimiter",      ":" },
          { "--preview-window", "top,50%,+{2}-/2" },
        },
        win_opts = {
          height = 50,
          width = 0.9,
          row = -2,
        },
    },
  })
```

**Comment by linrongbin16** [comment]
Let's continue this issue here: https://github.com/linrongbin16/fzfx.nvim/issues/710.
